Makefile

```diff
@@ -3,14 +3,24 @@
 
 PYTHON=`which python`
 
+# cli prefix for commands to run in container
```

Comment:
personally I would remove the makefile, and replace it with a proper README and/or a shell script.